gazebo_plugins/include/gazebo_plugins/gazebo_ros_camera.hpp

```diff
@@ -162,6 +162,15 @@ class GazeboRosCamera
   /// param[in] _enabled True to enable.
   void SetCameraEnabled(const bool _enabled);
 
+  // Get pointer to the cameras
+  const std::vector<gazebo::rendering::CameraPtr> GetCameras() const;
+
+  // Get camera name
+  const std::string GetCameraName() const;
```

Comment:
What advantage is there in returning const values here? I guess returning const references might save on some copying, but I think it's fine to return non-const values.

Reply:
That was to convey the design intent that these values are not meant to modified, but rather just be retrieved for reference. Let me know if you see no value in that and I can delete that specifier.

Reply:
I think he meant that you could return `const std::string&`?